delegates.json

```
@@ -54,4 +54,10 @@
         "url": "https://taostats.io",
         "description": "Supporting the bittensor eco-system through data provision, statistics and analytics."
     }
```

```suggestion
    },
```
It needs a comma here